phoenix-tracing-webapp/src/main/webapp/index.html

```diff
@@ -0,0 +1,114 @@
+<html ng-app="TracingAppCtrl">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Phoenix Tracing WebApp">
+    <meta name="author" content="nishani">
+    <link rel="icon" href="../../favicon.ico">
+
+    <title>Phoenix Tracing WebApp</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="css/bootstrap.css" rel="stylesheet">
+    <link href="/css/bootstrap-theme.css" rel="stylesheet">
+    <link href="css/nv.d3.css" rel="stylesheet">
+    <link href="css/font-awesome.css" rel="stylesheet">
+    <!-- Bootstrap theme -->
+    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
+    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
+    <script src="../../assets/js/ie-emulation-modes-warning.js"></script>-->
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <script src="js/lib/jquery-2.1.4.js"></script>
+    <script src="js/lib/bootstrap.js"></script>
+    <script src="js/sampleTrace.js"></script>
+    <script src="js/lib/d3.js"></script>
+    <!--angularjs -->
```

Comment:
- jquery is a widely used javascript library for simplifying client side scripting. MIT License
 - bootstrap is used for designing templates. MIT License (Apache License 2.0 prior to 3.1.0)
 - sampleTrace is a javascript library I have written. 
 - d3 charting library will be used to render charts. Sometimes it can be removed. BSD License